index.html

```diff
@@ -283,9 +283,9 @@ <h1 class="used-by">Used By</h1>
 	<section class="adopter-section">
 		<div class="adopter-container">
 			<h3 class="adopter-title">Add yourself as an <strong>adopter</strong></h3>
-			<p class="adopter-description">and share your service mesh story with the community</p>
+			<p class="adopter-description">Share your service mesh story with the community</p>
```

Comment:
For the time being, let's cherrypick this text update and proceed, while leaving the other CSS updates to the side for the time being.

The grayscale on the logos is intentional. The new design (in Figma) isn't necessarily better.

Reply:
Yeah Sure. I will undo the other changes.